### PR TITLE
added aws region in brig and galley in prod values example file

### PR DESCRIPTION
**Pull request by orandev**
Prod values example file was not updated after https://github.com/wireapp/wire-server-deploy/pull/197

### CODE REVIEW

**Comment by orandev** [comment]
You force the region in the fake aws charts:
_aws configure set region eu-west-1_
Shouldn't it be the same region in the prod values example file since it is fake-aws that is configured there?

**Comment by lucendio** [comment]
That is a good point, but `fake-aws` is somewhat independent from the example values files. Those files are meant to override defaults not defining them. 
Marking such a value as suggested above would explicitly point out that it needs review or at least should be questioned.

That said, in terms of `region` in `fake-aws`, it needs refactoring. In recent PRs it was started to make `region` more parameterize and less hard-coded. We need to pursue this path all the way to the end.

**Comment by julialongtin** [comment]
@lucendio can you link the issue that we need to solve before this can be merged / rejected?

**Comment by lucendio** [comment]
__tl;dr:__ I think we can merge this PR with a minor adjustment: @orandev can you please rollback to your [first commit](https://github.com/wireapp/wire-server-deploy/pull/229/commits/2f04ca9a3908f3b9c654017a5e477cb6571fc516). 

 #### Long story

The question, we need to ask here is, why are certain values set in the example values files and others are not? Well, when deploying Wire on Kubernetes the default assumption is an on-prem or private cloud installation - in any case, a scenario that is far away from AWS. 

Galley, Brig and Gundeck have all feasible default values for this scenario (fake-aws). But at the same time, the examples try to indicate where things need to be adjusted (`# change if using real AWS`), in case it a really runs on AWS. But sadly it's incomplete and inconsistent.

This change is a right step to address the latter. Follow-up work is tracked [here](https://github.com/zinfra/backend-issues/issues/1421)

**Comment by orandev** [comment]
> the default assumption is an on-prem or private cloud installation - in any case, a scenario that is far away from AW

We were advised to use real AWS on our on-prem platform so that is incorrect.

**Comment by lucendio** [comment]
> > the default assumption is an on-prem or private cloud installation - in any case, a scenario that is far away from AWS
> 
> We were advised to use real AWS on our on-prem platform so that is incorrect.

Of course, use cases may vary. I was referring to the __default__ perspective, when looking at the examples, and reference implementation that we are referring to in our [deployment guide](https://docs.wire.com/how-to/index.html) and aims to be cloud-agnostic.

One last request. Would you mind rebasing the branch against develop. after that, I'll will merge it!

**Comment by jschaul** [comment]
(we may wish to take multiple perspectives at some point: one being a production on-premise, and one being a production on-aws deployment. This requires that we first find a way to not have all of these value files drift apart so much though. Maybe we can make use of some templating. Or some abstraction around these values)
